### PR TITLE
various fixes

### PR DESCRIPTION
**Pull request by hummusonrails**
* Pointed to correct Docker image
* Added Couchbase VSCode extension to Codespace setup
* Removed Ruby IDE extensions from Gitpod setup
* Cleaned up env var instantiation in Codespace setup
* Migrated Actions workflow from a Java setup to a Node setup

### CODE REVIEW

File: .devcontainer/.devcontainer.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
     "name": "Couchbase [Framework] Workspace",
```
Comment (codercatdev) [inline .devcontainer/.devcontainer.json:2]: Should this be `Couchbase Node.js Workspace`?

File: .github/workflows/test-connection.yml [hunk 2]
```diff
@@ -35,14 +35,9 @@ jobs:
           source ${{ github.workspace }}/scripts/dbSetup.nu
           dbSetup $env.COUCHBASE_DEFAULT_BUCKET $env.COUCHBASE_DEFAULT_SCOPE $env.COUCHBASE_DEFAULT_COLLECTION
     - name: Set up JDK
```
Comment (codercatdev) [inline .github/workflows/test-connection.yml:37]: Set up Node

**Review by codercatdev — APPROVED** [review]
Couple minor updates for me to remember after merge 😉

File: .devcontainer/.devcontainer.json (hunk 1 above)
Comment (hummusonrails) [reply .devcontainer/.devcontainer.json:2]: Yes, good catch!

File: .github/workflows/test-connection.yml (hunk 2 above)
Comment (hummusonrails) [reply .github/workflows/test-connection.yml:37]: 💯

**Pull request merged**